README.md

```diff
@@ -15,10 +19,11 @@
 
 [Portkey's AI Gateway](https://portkey.ai/features/ai-gateway) is the interface between your app and hosted LLMs. It streamlines API requests to OpenAI, Anthropic, Mistral, LLama2, Anyscale, Google Gemini and more with a unified API. 
 
-✅&nbsp; Blazing **fast** (9.9x faster) with a **tiny footprint** (~45kb installed) <br>
+✅&nbsp; **Blazing fast** (9.9x faster compared to an ExpressJS server) with a **tiny footprint** (~45kb installed) <br>
```

Comment:
It dilutes the message, we should definitely include it in a benchmarks section. Should also talk about how we leverage core tech to make the gateway faster along with launching the perf benchmarks that @VisargD is preparing

Reply:
Hey @roh26it @vrushankportkey - The benchmark might take some time. Can we merge this PR separately?